settings/package.json

```diff
@@ -15,7 +15,8 @@
 		"test:unit": "wp-scripts test-unit-js"
 	},
 	"dependencies": {
-		"@automattic/generate-password": "^0.1.0"
+		"@automattic/generate-password": "^0.1.0",
+		"lodash": "^4.17.21"
```

Comment:
> Or was it added to document that it's used?

Yeah, it was added because of linting error
`10:1 error 'lodash' should be listed in the project's dependencies. Run 'npm i -S lodash' to add it  import/no-extraneous-dependencies`

> IIRC there's a way to do that.

Perhaps something like this [one](https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/js/function-reference.js#L1)? 

Unfortunately, adding that comment doesn't fix the error, I've checked how Gutenberg handled it, and they just put it to `devDependencies`([ref](https://github.com/WordPress/gutenberg/blob/trunk/package.json#L205)) (The version is different from core, and they seem to [manually](https://github.com/WordPress/gutenberg/commit/d37bcb48d7028aa7701ff129ad6e338c65d1666f) updated it)

I've tried moving it to `devDependencies` and it solved the linting error, but I guess it's ok not to add it.

Reply:
I agree with the `devDependencies` approach.